public/app/plugins/datasource/loki/result_transformer.ts

```diff
@@ -246,7 +245,7 @@ export function lokiResultsToTableModel(
   table.columns = [
     { text: 'Time', type: FieldType.time },
     ...sortedLabels.map((label) => ({ text: label, filterable: true, type: FieldType.string })),
-    { text: resultCount > 1 || valueWithRefId ? `Value #${refId}` : 'Value', type: FieldType.number },
+    { text: `Value #${refId}`, type: FieldType.number },
```

Comment:
I like this! It can be slight breaking change, but I think it is worth it. Maybe we could add a breaking change note (even though it is tiny breaking change, but it would be nice to have it documented)

Reply:
i actually think it's not even a breaking change...probably.. if you check this commit https://github.com/grafana/grafana/pull/47571/commits/6c6243340d5bb24f3d54b37b7396bc7f6ecc124c  :  
- i checked, and everytime the function `lokiResultsToTableModel` is called, the last param `valueWithRefId` is always `true`. so we can remove that param, and simply replace `valueWithRefId` with `true` in the function-body
- so this code ``resultCount > 1 || valueWithRefId ? `Value #${refId}` : 'Value'``
- becomes : ``resultCount > 1 || true ? `Value #${refId}` : 'Value'``
- but `X OR true` is always `true`
- so the code becomes ``true ? `Value #${refId}` : 'Value'``
- which is just `Value #${refId}`

at least i hope so, maybe i missed something...